### ISSUE TITLE
YOUR-CABIN-API-KEY

### ISSUE THREAD

**Issue by wanglong167** [cabinjs/cabin#130]
YOUR-CABIN-API-KEY

**Comment by wanglong167** [comment]
// (optional: your free API key from https://cabinjs.com)

    server Open source?

**Comment by wanglong167** [comment]
@niftylettuce

**Comment by armenr** [comment]
...So, believing the documentation, I went ahead and implemented the vanilla boilerplate to test out the logging...(since it worked in Dev without the optional API_KEY...so why not?!).

Broke my production servers :)

Where do we get said API key? Looks like Axe references a self-hosted "API", but the link to that is broken and there's no evidence of that repository on GitHub.

...I'm wondering, is Cabin even technically usable, in general?

**Comment by niftylettuce** [comment]
If you are using in production right now, please set `capture: false` as an option when you initialize Cabin.

The API keys will be available soon, right now I am finishing up Cabin's front-end.  It's amazing and fast, and will be completely open-source too.  In the meanwhile, all of Cabin's open-source tools and packages (including Axe for example) work 100% perfectly.

**Comment by armenr** [comment]
Thank you @niftylettuce for ALL of your work! I'll set that to `false` and try again. As an Ops guy of 10 years gone DevOps, and now just Dev-ing, please allow me to say what an excellent library and logging toolchain you've set up. 

People like you make OpenSource awesome. :)

**Comment by gperdomor** [comment]
@niftylettuce can you provide any ETA or some screenshots please? I can't wait 🤓 

I would like to use it in a new project 😃

**Comment by armenr** [comment]
^^ I second this.

**Comment by niftylettuce** [comment]
I am trying as best and fast I can.  I also am simultaneously spread thin between running OurHarvest.com and ForwardEmail.net too.  I will keep you guys posted asap.  Just got Lad to a beta release, see https://lad.js.org.

**Comment by gperdomor** [comment]
any update about this?

**Comment by niftylettuce** [comment]
We will be launching our logging service in the near future.  I'm going to keep this open for now.

**Comment by niftylettuce** [comment]
In the meanwhile, if you want to collect logs on your server, you can reference how we did it in Lad at <https://github.com/ladjs/lad.sh/blob/master/app/controllers/api/v1/log.js> and <https://github.com/ladjs/lad.sh/blob/066427cffe9f6538ff4c97b90e762840b299008e/assets/js/logger.js#L8> and <https://github.com/ladjs/lad.sh/blob/066427cffe9f6538ff4c97b90e762840b299008e/routes/api/v1/index.js#L10>.

**Comment by armenr** [comment]
YESSSSSS @niftylettuce !

**Comment by Nicholaiii** [comment]
Any update on this?

**Comment by JustDoItSascha** [comment]
I would also interested in updates? Is this project actively developed? Maybe I'm stupid, but I don't understand where I can see the logs?

**Comment by niftylettuce** [comment]
Yes it's 100% in development and beta.  I am very close to finishing, I just need a little more time.

My project https://forwardemail.net/open-startup really took off as did my day job, so I've only been able to dedicate time on weekends, but will be fueled by a lot of coffee and will try to get to the finish line for all of you.  Will definitely give everyone here 2-3+ years of free usage.

**Comment by jferg-newcontext** [comment]
Definitely understand and can sympathize with being busy - any chance this info could to into the main web page to save folks some time , since it's definitely not clear from the docs?

**Comment by ElectronSz** [comment]
So what is the plan?

**Comment by armenr** [comment]
@niftylettuce - Thanks for the promise you made (this isn't passive aggressive, I really mean it)! Your new service looks super cool! My startup is actually sorta interested to talk to you about it re: some questions/features we wanna ask about (not to request for you to add them, but more specifically about those you already have).

**Comment by niftylettuce** [comment]
@armenr Feel free to email me at niftylettuce@gmail.com!

**Comment by titanism** [comment]
See Axe/Cabin v10 with updated README examples for sending to HTTP endpoints, Sentry, and more.
